views/rodauth/verify-first-account-email.erb

```diff
@@ -1,3 +1,11 @@
-Someone has created an account with this email address. If you did not create this account, please ignore this message.
+Hello!
 
-If you created this account, please go to <%= rodauth.verify_account_email_link %> to verify the account and specify a password.
\ No newline at end of file
+Your Osso instance is deployed and ready to start using.
+
+Please visit <%= rodauth.verify_account_email_link %> to create your admin account and specify a password. You'll be able to invite teammates to your instance once you've signed in.
+
+Review our documentation for integrating Osso into your application.
```

Comment:
@sbauch is this meant to include a link to docs?

Reply:
prob should huh, ill get it sorted